nautobot/circuits/tests/test_filters.py

```diff
@@ -37,7 +37,6 @@ def setUpTestData(cls):
 
         providers = Provider.objects.all()[:2]
         circuit_types = CircuitType.objects.all()[:2]
-        cls.regions = Region.objects.filter(sites__isnull=False, children__isnull=True, parent__isnull=True)[:2]
```

Comment:
The changes to this file aren't from `develop`, they're just some leftovers I noticed while troubleshooting merge conflicts.